signadot/operator/values.yaml

```diff
@@ -59,12 +65,17 @@
 # linkerd:
 #   # Enable the integration with Linkerd
 #   enabled: true
-
-
+#   operator:
+#     # Specify custom pod annotations for operator workloads placed on the Linkerd mesh. 
+#     # If the field does not exist, or no annotations are specified, then it will default to 
+#     # linkerd.io/inject: "enabled"
+#     annotations:
+#       linkerd.io/inject: enabled
+#       config.linkerd.io/proxy-await: enabled
 # Component settings
 # -----------------------------------------------------------------------------
 
-operator:
+controllerManager:
```

Comment:
We should reflect this change in the doc.

Reply:
@scott-cotton, there are still some references to `operator.xxx`:

![Screenshot from 2025-05-20 10-11-01](https://github.com/user-attachments/assets/5979f84a-c720-4f5a-bd02-4bc8b14fff29)

![Screenshot from 2025-05-20 10-11-21](https://github.com/user-attachments/assets/672e347e-a46d-4465-bcdf-b972481fe86b)